### PR TITLE
changing columns settings and ordering in deliveries table for donor mode only

### PR DESCRIPTION
**Pull request by morfi11**

### CODE REVIEW

**Comment by morfi11** [comment]
לא טוב?

File: src/app/family-deliveries/family-deliveries.component.ts [hunk 1]
```diff
@@ -636,15 +636,29 @@ export class FamilyDeliveriesComponent implements OnInit, OnDestroy {
     this.refreshStats();
     let cols = this.deliveries.columns;
     let firstColumns = [
-      cols.items[0],
```
[the remark this answers is not in the document]
Comment (morfi11) [reply src/app/family-deliveries/family-deliveries.component.ts:639]: agree but this was your change before... how do you recommend to do it instead?

File: src/app/family-deliveries/family-deliveries.component.ts [hunk 2]
```diff
@@ -468,7 +468,10 @@ export class FamilyDeliveriesComponent implements OnInit, OnDestroy {
           }
         },
         deliveries.basketType,
-        deliveries.quantity,
+        {
```
[the remark this answers is not in the document]
Comment (morfi11) [reply src/app/family-deliveries/family-deliveries.component.ts:471]: this specific change should not be a problem - just reducing the width of a numeric field that is just too wide now anyway

File: src/app/family-deliveries/family-deliveries.component.ts [hunk 3]
```diff
@@ -456,7 +456,7 @@ export class FamilyDeliveriesComponent implements OnInit, OnDestroy {
 
         {
           column: deliveries.name,
-          width: '200'
+          width: '160'
```
[the remark this answers is not in the document]
Comment (morfi11) [reply src/app/family-deliveries/family-deliveries.component.ts:459]: can add the "if" here too, or modify the width later below (can we?). But actually, reducing the width should not be bad for anyone else... 
let me know what you prefer

**Comment by noam-honig** [comment]
What makes sense in the context of the first 5 - doesn't make in the
context of 33.

See sortColumns in Families for something similar to what you're looking
for - refactor sortColumns to a shared location and do that.
Or - tell me which columns you want and I'll see what I can do
*Noam Honig  *
*Founder & **CEO*
*www.fireflymigration.com <http://www.fireflymigration.com/>*
[image: Inline image 1]
[image: Inline image 5] <http://www.youtube.com/fireflymigration>[image:
Inline image 4]
<http://www.linkedin.com/company/firefly-migration/100-automated-migration-from-magic-unipaas-xpa-to-net-23400981/product>

**Comment by noam-honig** [comment]
I diss agree on the width - I prefer to see more text in most cases, in
this specific case you may leave the 160 - but not for all - for the rest
do an if or something similar
*Noam Honig  *
*Founder & **CEO*
*www.fireflymigration.com <http://www.fireflymigration.com/>*
[image: Inline image 1]
[image: Inline image 5] <http://www.youtube.com/fireflymigration>[image:
Inline image 4]
<http://www.linkedin.com/company/firefly-migration/100-automated-migration-from-magic-unipaas-xpa-to-net-23400981/product>

**Comment by morfi11** [comment]
reopening to allow Aiham add his modified code

File: sample.env [hunk 4]
```diff
@@ -1,36 +0,0 @@
-DATABASE_URL='postgres://postgres:MASTERKEY@localhost/tributest'
```
Comment (morfi11) [inline sample.env:1]: why was this deleted?

File: src/app/app.component.ts [hunk 5]
```diff
@@ -38,6 +38,7 @@ export class AppComponent {
     private helper: RouteHelperService,
     public context: Context,
     public settings: ApplicationSettings) {
+      console.log(settings)
```
Comment (noam-honig) [inline src/app/app.component.ts:41]: please remove that :)
